src/app/containers/Caption/index.jsx

```diff
@@ -10,40 +11,61 @@ import InlineLink from '../InlineLink';
 
 const componentsToRender = { fragment: Fragment, urlLink: InlineLink };
 
+const chooseOffscreenText = (mediaType, videoCaption, imageCaption) => {
+  let offscreenText = '';
+  if (mediaType === 'video') {
+    offscreenText = videoCaption;
+  } else if (mediaType === 'image') {
+    offscreenText = imageCaption;
+  } else {
+    offscreenText = false;
```

Comment:
In this case, the value of the offscreenText will be `false`, which we likely do not want this to display the value on the page. 

We should have unit tests to capture each case - if it's of type `video`, `image` or type is not defined.  

If visually-hidden span is empty, screenreader users would think that the caption is part of the main article body. 

Having talked with @greenc05 about this, we should have a default `captionOffscreenText` which should have the value `Caption, ` for the News service. This should be used in the case where type is not a known type such as `image` or `video`.